src/main/java/classify/commands/DeleteCommands.java

```diff
@@ -61,17 +65,28 @@ public static void restoreStudent(ArrayList<Student> masterStudentList, ArrayLis
             studentName = promptName(in);
         }
 
-        Student foundStudent = InputParsing.findStudentByName(recentlyDeletedList, studentName);
+        Student foundStudent = InputParsing.findStudentByName(recentlyDeletedList, studentName, in);
 
         if (foundStudent != null) {
             UI.printRestoreMessage();
         } else {
             UI.printStudentNotFound();
         }
 
-        UI.printDivider();
-        masterStudentList.add(foundStudent);
-        recentlyDeletedList.remove(foundStudent);
+        //@@author Cryolian
```

Comment:
Awesome way of allowing the user to add more than 1 student of the same name! Great idea.